### PR TITLE
update crio.conf man page.

### PR DESCRIPTION
**Pull request by Chenditang**
**- What I did**
update crio.conf man page.

**- How I did it**

**- How to verify it**

### CODE REVIEW

**Comment by rhatdan** [comment]
Thanks @Chenditang 
LGTM

File: docs/crio.conf.5.md [hunk 1]
```diff
@@ -145,6 +151,9 @@ The `crio.runtime` table contains settings pertaining to the OCI runtime used an
 
     2) `/usr/share/containers/mounts.conf`: This is the default file read for mounts. If you want CRI-O to read from a different, specific mounts file, you can change the default_mounts_file. Note, if this is done, CRI-O will only add mounts it finds in this file.
 
+**bind_mount_prefix**=""
```
Comment (TomSweeneyRedHat) [inline docs/crio.conf.5.md]: maybe change to '="prefix" ' to let the user know a value is necessary.

File: docs/crio.conf.5.md [hunk 2]
```diff
@@ -145,6 +151,9 @@ The `crio.runtime` table contains settings pertaining to the OCI runtime used an
...
+  A prefix to use for the source of the bind mounts.  This option would be useful if you were running CRI-O in a container.  And had / mounted on /host in your container.  Then if you ran CRI-O with the --bind-mount-prefix=/host option, CRI-O would add /host to any bind mounts it is handed over CRI.  If Kubernetes asked to have /var/lib/foobar bind mounted into the container, then CRI-O would bind mount /host/var/lib/foobar.  Since CRI-O itself is running in a container with / or the host mounted on /host, the container would end up with /var/lib/foobar from the host mounted in the container rather then /var/lib/foobar from the CRI-O container.
```
Comment (TomSweeneyRedHat) [inline docs/crio.conf.5.md]: 'container.  And' to "container and '

Comment (TomSweeneyRedHat) [reply docs/crio.conf.5.md]: Maybe it's me having Friday afternoon brain, but I'm not sure what "had / mounted on /host".  Does this mean "had the '/' on the host mounted as '/host' on  your container?

**Comment by rhatdan** [comment]
@Chenditang Are you still working on this?

**Comment by umohnani8** [comment]
@Chenditang are you still working on this? If so, please rebase and fix the CI errors.

**Comment by rhatdan** [comment]
@umohnani8 Could you finish this one up for @Chenditang?

**Comment by umohnani8** [comment]
@rhatdan sure.

**Comment by umohnani8** [comment]
Closing in favor of #2648

**Pull request closed without merging**